### ISSUE TITLE
Example not working

### ISSUE THREAD

**Issue by BR88C** [discordjs/voice#106]
Couldn't think of a good title

When using the example code as written, from my experience it would not play without waiting for the resource to be ready.

[Original Code](https://github.com/discordjs/voice/blob/1b1a21f0b0c4111ed5e484e6345934bfd13543f1/examples/basic/basic-example.ts#L46)
```js
player.play(resource)
```

Modified line
```js
resource.playStream.once(`readable`, () => player.play(resource));
```

I would submit a PR to modify the example, however I'm unsure if this is an edge case due to some unknown cause.

**Further details:**

- @discordjs/voice version: `0.2.0`
- Node.js version: `14.17.0`
- Operating system: `Windows 10 Pro Build 19042`
- Priority this issue should have: Unsure, as this could be an edge case.

**Relevant client options:**

Using `discord-rose` at master.

**Comment by amishshah** [comment]
Hey!

This is indeed a bug, it's fixed in https://github.com/discordjs/voice/commit/686ef198d1e6a56061c263b7073d396e03e572af however this is not released as of yet. I am planning to make a release during this week which will include that commit, alongside a more thorough example of using this library :)

Keeping this issue open until then.

**Comment by amishshah** [comment]
Have actually released this separate to the example, should be available now in [v0.2.1](https://www.npmjs.com/package/@discordjs/voice/v/0.2.1)
